<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientTreeAreaDisplay</class>
 <widget class="QWidget" name="ClientTreeAreaDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>1107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="useDomainMap">
       <property name="text">
        <string>Domain Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqSourceComboBox" name="domainMap"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Tree Layout</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Layout Strategy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="layoutStrategy"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Shrink Percentage</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="shrinkPercentage">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="vertices">
     <property name="title">
      <string>Area</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Labels</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="areaLabels">
           <property name="text">
            <string>Visible</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="areaLabelArray"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Priority Array</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="areaLabelPriorityArray"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Hover Array</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="areaLabelHoverArray"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Font Size</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="areaLabelFontSize"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Color</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="areaColorByArray">
           <property name="text">
            <string>Array</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="areaColorArray"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="pqColorChooserButton" name="areaColor">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="areaSizeArray"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="edges">
     <property name="title">
      <string>Edges</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Labels</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="edgeLabels">
           <property name="text">
            <string>Visible</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="edgeLabelArray"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Font Size</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="edgeLabelFontSize"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Color</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="edgeColorByArray">
           <property name="text">
            <string>Array</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="edgeColorArray"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="pqColorChooserButton" name="edgeColor">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bundling Strength</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="edgeBundlingStrength"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>151</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QPushButton</extends>
   <header>pqColorChooserButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqSourceComboBox</class>
   <extends>QComboBox</extends>
   <header>pqSourceComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
